File: 0242/main.py
from typing import Dict


class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False

        hashmap: Dict[str, int] = {}
        value = ""
        count = 0
        for value in s:
            hashmap[value] = hashmap.get(value, 0) + 1
        for value in t:
            count = hashmap.get(value, 0)
            if count > 0:
                hashmap[value] = count - 1
            else:
                return False

        return True

File: 0242/main.ts
function isAnagram(s: string, t: string): boolean {
    if (s.length != t.length) {
        return false
    }
    let hashmap: Map<string, number> = new Map()
    let value = ""
    let count = 0
    for (value of s) {
        hashmap.set(value, (hashmap.get(value) ?? 0) + 1)
    }
    for (value of t) {
        count = hashmap.get(value) ?? 0
        if (count > 0) {
            hashmap.set(value, count - 1)
        } else {
            return false
        }
    }
    return true
};